server/routerlicious/packages/test-utils/src/testTenantManager.ts

```diff
@@ -71,7 +75,7 @@ export class TestTenantManager implements ITenantManager {
 		return this.tenant;
 	}
 
-	public async getTenantGitManager(tenantId: string, documentId: string): Promise<GitManager> {
+	public async getContainerGitManager(tenantId: string, documentId: string): Promise<GitManager> {
```

Comment:
You can't do this yet, because ITenantManager comes from R11s. This change will need to happen when T9s consumes a newer version of R11s packages with the rest of the changes in this PR.

Reply:
Will add T9 person when publish the official PR